### ISSUE TITLE
Reporting "Comments" and "Notes" in Excel

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#9193]
### Steps to reproduce:

TL;DR: In Office 365 at least, It looks like Microsoft have changed what we know as comments to "notes", and added a new thing called Comments (with more reply options).  This causes confusion in identification and reading, as demonstrated below.  Note this issue is purely around the naming, not about the problems executing Part B or reporting the comment in Part C).

Part A)
1. Open a blank Excel spreadsheet.
2. Press SHIFT+F2 to open the Comment dialog.
3. Add a comment and press OK.
4. Arrow away from and back to the cell.  NVDA reports "Has comment".
5. Press the APPLICATIONS KEY to open the context menu.  Options include "Edit note", "Show / Hide note" and "delete note".

Part B)
1. On another cell, open the context menu.
2. Select "New note".
4. NVDA stops talking. Type something.
5. Press ESCAPE twice to save the note and return to Excel.
6. NVDA freezes (see issue #9194), press ALT+TAB to another window and back again.
7. Move away from the cell and back.  NVDA reports "Has comment".
8. Options in context menu include "Edit note", "Show / hide note" and "Delete note".

Part C)
1. On another cell, open the context menu.
2. Select "New comment".
3. Type a comment, tab to "Post" button and press ENTER.
4. The comment is still visible, press ESCAPE to close.
5. Arrow away and back to the cell, NVDA does not report the comment (see issue #9195).
6. Open the context menu, options are present to "Reply to comment" and "Delete comment"

### Actual behavior:

Part A adds what NVDA refers to as a comment, but Excel refers to as a note.
Part B is an alternate method to add the same thing as Part A.
Part C adds what Excel refers to as a note, but NVDA does not report.

### Expected behavior:

The feature used in Parts A and B should be referred to as a "Note" rather than a comment.
The feature used in part C should be referred to as a "Comment".

(I believe the difference is that comments can be added by different users and can be replied to with the author of each comment attributed.  Notes are attributed to their author, but can't be "replied to".

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2018.4.1
#### Windows version:
10 build 18272.1000 
#### Other information about your system:
Office 365 version 16.0.11126.20200 

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by Adriani90** [comment]
I think it is better to discuss wording on #9195 which would deal with implementation of reporting of new style comment at all. If #9195 is implemented, then also the comment editing dialog "shift+f2" should be renamed to "note editing dialog".
I am closing this for now. Let's continue discussions on this on #9195.
